desk/desk.bill

```diff
@@ -7,5 +7,6 @@
     %groups-ui
     %channels
     %channels-server
+    %activity
```

Comment:
I think we should probably leave it out and turn it on manually until we're ready for it to run in production. This won't be released for a while but it would be nice to let it flow through the release pipeline, but not be on by default.